src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useState } from 'react/cjs/react.production.min';
+import ToDoList from './ToDoList';
+
+function App() {
+  const [todo, setTodo] = useState([]);
+  const [newTask, setNewTask] = useState('');
+
+  function onNewTask(event) {
+    setNewTask(event.target.value);
+  }
+
+  function onAddTask() {
+    setTodo([...todo, newTask]);
+    setNewTask('');
+  }
+
+  function onRemoveTask(index) {
+    setTodo(todo.filter((item, i) => i !== index));
+  }
+
+  return (
+    <>
+      <h1>To-do</h1>
+      <input placeholder="할 일을 입력해 주세요" value={newTask} onChange={onNewTask} />
```

Comment:
저는 속성이 2개이상인 경우 코드가독성을 고려해서 줄바꿈을 해주는편이에요.
비교했을 때 어떤가요?

```suggestion
      <input
         placeholder="할 일을 입력해 주세요"
         value={newTask}
         onChange={onNewTask}
       />
```

Reply:
프리티어 쓰던 습관이 남아있어서 확인을 못했네요
저도 줄바꿈 한쪽이 훨씬 좋습니다

done: [1b60237](https://github.com/CodeSoom/react-week2-assignment-2/pull/118/commits/1b602378f0f4c4fea6145b1443313c176438e9d1)

Reply:
린트에서 코드 스타일을 대부분 잡아주기 때문에 프리티어와 함께 사용할 이유가 굳이 없어요!